dressup/src/components/Categories/Categories.js

```diff
@@ -0,0 +1,14 @@
+import "./Categories.css";
+
+function Categories() {
```

Comment:
suggestion: use es6+ functions like " const Categories = () => { ... } "

Reply:
The files and folder names are inconsistent in naming like you used Searchbar.js, Topbrandssection.js, HeroSectionbutton.js. Name components like " SearchBar.js, TopBrandsSection.js, HeroSectionButton.js " and CSS files like " search-bar.css, top-brands-section.css, hero-section-button.css " and all folder names like either " Buttons, Components, TopBrandsSection  " or " buttons, components, top-brands-section ".